include/enums.h

```diff
@@ -5983,50 +5984,73 @@ enum FoldStateFlags {
 };
 
 typedef enum FoldType {
-    FOLD_TYPE_NONE                    = 0x0,
-    FOLD_TYPE_1                       = 0x1,
-    FOLD_TYPE_2                       = 0x2,
-    FOLD_TYPE_3                       = 0x3,
-    FOLD_TYPE_4                       = 0x4,
-    FOLD_TYPE_5                       = 0x5,
-    FOLD_TYPE_6                       = 0x6,
-    FOLD_TYPE_7                       = 0x7,
-    FOLD_TYPE_8                       = 0x8,    // color overlay?
+    FOLD_UPD_CLEAR                    = 0x0,    // or FOLD_UPD_INIT?
+    FOLD_TYPE_1                       = 0x1,    // unused?
```

Comment:
Why UPD (update?) over TYPE now?

Reply:
These are commands consumed by `fold_update`. Originally, I believed each FOLD_TYPE was associated with a different 'paper' animation, but now its clear that only FOLD_UPD_SET_ANIM (and to some extent FOLD_UPD_WAVY) does this. Instead, most of the "FOLD_UPD" commands are used to adjust color and alpha blending parameters for the objects they're applied to, with fairly subtle distinctions between them.